docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1314,6 +1314,62 @@ See the proofs for this approach below.
 
 .. image:: baseline_calibration_proofs.png
 
+Summary of Iron Intervention Algorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+  # Definitions
+  t         := current time in years since start of simulation
+  t_end     := time when simulation ends, in years since start (e.g. 6 years)
+  20_weeks  := 20 weeks, measurd in years (e.g. 20*7/365.25)
+
+  # Population level parameters
+  hb_shift              = effect size on hemoglobin (normally disributed, mean 3.0 g/L)
+  bw_response           = dose-response on birthweight (normally distributed, mean 15.1/10 g/mg)
+  baseline_coverage(t)  = iron fortification coverage at time t in baseline scenario
+  delta_coverage(t)     = change in coverage from baseline to intervention
+  coverage(t)           = iron fortification coverage at time t
+                        = baseline_coverage(t) + delta_coverage(t)
+    # Multiplier on hb_shift due to age
+  hb_age_fraction(age)  = 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+    # Multiplier on hb_shift due to lag in response time (=1 if simulant hasn't yet received fortification)
+  hb_lag_fraction(time_since_coverage)
+                        = time_since_coverage/0.5 if 0<=time_since_coverage<0.5 else 1
+  iron_concentration    = concentration of iron in the country's fortified flour
+
+  # Individual level attributes
+  p_i = propensity of simulant and mother for exposure to unfortified flour
+  daily_flour_i = average daily flour consumption of simulant's mother
+  gestational_age_i = gestational age of simulant drawn from GBD (in years)
+  birthweight_gbd_i = birthweight drawn from GBD for simulant
+  age_i(t) = the simulant's age at time t
+  hb_gbd(age_i(t)) = hemoglobin level drawn from GBD for simulant
+  birthweight_i := simulant's birthweight after adjusting for mother's iron consumption
+  hb_i(t) := simulant's hemoglobin level after adjusting for fortification
+
+  # Stratification on child coverage - assign categories on simulant initialization
+  child_fortified_baseline_end_i = p_i < baseline_coverage(t_end)
+  child_fortified_end_i = p_i < coverage(t_end)
+
+  # At beginning of simulation
+  mother_fortified_i = "unknown"
+  mother_fortified_baseline_i = "unknown" # For stratification
+  birth_weight_i = birthweight_gbd_i
+  child_fortified_i(0) = p_i < coverage(0)
+  hb_i(0) = hb_gbd(age_i(0)) + (-coverage(0)+child_fortified_i(0)) * hb_shift *  hb_age_fraction(age_i(0))
+
+  # When simulant is born in the simulation
+  mother_fortified_i = p_i < coverage(t - gestational_age_i + 20_weeks)
+  mother_fortified_baseline_i = p_i < baseline_coverage(t - gestational_age_i + 20_weeks) # For stratification
```

Comment:
We might need to put a note here about what to do if a baby has a gestational age less than 20, which should be rare, but is possible based on our exposure distribution.

Reply:
Good point. Do we also need to worry about making sure the mother gets iron for at least 7 weeks? E.g. if they are born between 20 - 27 weeks.  I'll add a piece to try to handle those situations.

Reply:
@alibow So basically, should we just say that `mother_fortified_i = False` in either of the two situations above, i.e. if the gestational age is less than 27?  What if the mother had been eating iron-fortified food for the entire pregnancy up to that point?  Is it only the critical window 20-27 weeks that counts, or would, e.g. 19-26 weeks count as well?

Reply:
Hmm, I'm not quite sure about that, but based on the Haider studies (https://drive.google.com/drive/u/1/folders/1X3Mx8mmPBuAi7wjTMppWuoSjxiZ64-S0), I think we should not require that the critical range is 20-27 weeks, but rather than fortification exposure needs to occur at least before 20 weeks of gestation (although we could go up to 28 weeks, which is the latest start in the Haider review) and require that duration of exposure before birth is at least 7 weeks.